index.js

```diff
@@ -60,15 +66,38 @@ renderPlayer(player)
 
 
 /***** Deliverable 1 *****/
-function toggleColor(element) {
+const toggleColor = (element) => {
   if (element.style.color === "red") {
     element.style.color = "black"
   } else {
     element.style.color = "red"
   }
+  // console.log(element)
 }
 
+header.addEventListener('click', e => {
+  if (e.target.textContent === header.textContent) {
+    toggleColor(header)
+  }
+})
+
+
 /***** Deliverable 2 *****/
+let like = 0+1
+
+playerCont.addEventListener('click', e => {
+  if (e.target.matches('button')) {
+    likes.textContent = `${like++} Likes`
+  }
+})
 
 
 /***** Deliverable 3 *****/
+
+newGoal.addEventListener('submit', e => {
+  e.preventDefault()
+  const description = newGoal.description
+  const link = newGoal.link
+  
+  
```

Comment:
You're on the right track for getting the values from the form - you can either just re-write the code for creating the DOM elements, or if you want to use the renderGoal function, you can create an object (since renderGoal needs to be called with an object):

```js
This isn't quite right, since the like variable isn't referencing the player's likes - it's just a number you made separately. You'd want to either use `player.likes` or `parseInt(likes.textContent)` to get the current number of likes, and increment it.

Also, you don't need event delegation here since you're only listening for events for one button (it works, but is extra code you don't necessarily need).

Here's a working solution:

```js
  const description = newGoal.description
  const link = newGoal.link
  const goalObj = {
    description: description,
    link: link,
    playerId: player.id
  }

  renderGoal(goalObj)
```